startup/00-nsls2-tools.py

```diff
@@ -44,9 +44,10 @@ def wait_for_connection(self, timeout=DEFAULT_CONNECTION_TIMEOUT):
 from IPython import get_ipython
 from bluesky.utils import PersistentDict
 from pathlib import Path
+from csx1.analysis.callbacks import BECwithTicks
 
 ip = get_ipython()
-nslsii.configure_base(ip.user_ns, 'csx', publish_documents_with_kafka=True)
+nslsii.configure_base(ip.user_ns, 'csx', publish_documents_with_kafka=True, bec=False)
```

Comment:
what is the impact of `bec = False`?  is it only affecting LivePlot ?  Does this remove the possibility of using`bec.peaks` or `LivePlotPlusPeaks`?


if the issue is that someone decided that sharex and sharey should be True and resulted in this custom piece of code below `class BECwithTicks(BestEffortCallback)` then I wonder if that shouldn't be discussed on a project level - atleast to understand why and to add something to the main project so this bit of code doesn't exist in a thousand places

Reply:
Inside `nslsii.configure_base` it creates a `bec = BestEffortCallback()`, and then adds that to the namespace and subscribes the RunEngine to it if `bec = True`. 
So the two changes in coordination say, don't let  `nslsii.configure_base` create my bec, I'll make it myself.

Reply:
As for the project level, I think in most when sharing the x/y axes it is customary to not redundantly label them with text. In your case where there are many large plots and it's hard to track, the redundancy becomes effective.